wptdash/blueprints/routes.py

```diff
@@ -73,6 +75,79 @@ def job_detail(job_number):
                            org_name=ORG, repo_name=REPO)
 
 
+@bp.route('/performance')
+def performance_metrics():
+
+    def get_quarter_start_date(quarter, year):
+        quarter_first_month = quarter * 3 - 2
+        quarter_start = date(year, quarter_first_month, 1)
+        return quarter_start
+
+    def get_default_start_end():
+        month_end_days = {
+            3: 31,
+            6: 30,
+            9: 30,
+            12: 31
+        }
+        today = date.today()
+        month = today.month
+        year = today.year
+        quarter = math.ceil(month / 3)
+        quarter_start = get_quarter_start_date(quarter, year)
+        delta = today - quarter_start
+        if delta.days < 45:
+            quarter -= 1
+            if quarter == 0:
+                quarter = 4
+                year -= 1
+            quarter_start = get_quarter_start_date(quarter, year)
+        quarter_end_month = quarter_start.month + 2
+        quarter_end = date(year, quarter_end_month,
+                           month_end_days[quarter_end_month])
+        return quarter_start, quarter_end
+
+    db = g.db
+    models = g.models
+    (default_start, default_end) = get_default_start_end()
+    start_date = request.args.get('start') or default_start
+    end_date = request.args.get('end') or default_end
+    okr_delta = timedelta(minutes=30)
```

Comment:
Putting some internal Google thing directly in the code seems weird here. I would prefer a histogram that could be used to compute this, and also tell us about outliers, etc.

Reply:
A histogram would be neat. No need to make it pretty and graphical though, that seems like too much effort :)

Reply:
I'll whip something up.